.github/workflows/registry-docs-pr-based.yml

```diff
@@ -0,0 +1,174 @@
+on:
+  workflow_call:
+    inputs:
+      providerFqn:
+        description: "Provider FQN to convert, e.g. hashicorp/aws"
+        required: true
+        type: string
+      additionalProviderFqns:
+        description: "Additional provider FQNs that are required to convert because they are used in examples, e.g. 'hashicorp/azurerm,hashicorp/google'"
+        required: false
+        type: string
+      maxRunners:
+        description: "Maximum number of parallel runners to use"
+        required: false
+        default: 10
+        type: number
+      gitUser:
+        description: "Git user to commit as"
+        required: false
+        default: "team-tf-cdk"
+        type: string
+      gitEmail:
+        description: "Git email to commit as"
+        required: false
+        default: "github-team-tf-cdk@hashicorp.com"
+        type: string
+      parallelFileConversions:
+        description: "Number of files to convert in parallel"
+        required: false
+        default: 1
+        type: number
+      parallelConversionsPerDocument:
+        description: "Number of conversions to run in parallel per document"
+        required: false
+        default: 3
+        type: number
+      languages:
+        description: "Languages to convert to as comma-separated list, e.g. 'typescript,python'"
+        required: false
+        default: "typescript,python"
+        type: string
+      files:
+        description: "Files to convert as comma-separated glob list, e.g. '*/ec2_*.html.markdown,*/s3_*.html.markdown'"
+        required: false
+        default: "**/*"
+        type: string
+      cdktfRegistryDocsVersion:
+        description: "Version of cdktf-registry-docs to use"
+        required: false
+        default: "1.12.0"
+        type: string
+    secrets:
+      GH_PR_TOKEN:
+        description: "GitHub token to create PRs with"
+        required: true
+
+name: CDKTF Provider Documentation
+jobs:
+  cdktfDocsSetupBranch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - run: git config --global user.email "${{ inputs.gitEmail }}"
```

Comment:
Was this on purpose, instead of `- run: |`

Reply:
Doesn't really matter to me :D But yeah could be combined :)